### PR TITLE
Add the sample model to describe that the interface between transform code and model definition is tensors.

### PR DESCRIPTION
**Pull request by brightcoder01**
Relate #1885 

Model definition from model zoo
```python
def wide_and_deep_classifier(input_layers, wide_embedding, deep_embedding):
```

Generated transform python code
```python
def transform_from_code_gen(source_inputs):
```

### CODE REVIEW

File: model_zoo/census_model_sqlflow/wide_and_deep/wide_deep_functional_tensor_interface_fc.py [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+        )
+
+    return input_layers
+
+
+# Build the transform logic from the metadata in feature_configs.py.
+def transform(inputs):
+    feature_column_dict = {}
+
+    for feature_transform_info in FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY:
```
Comment (typhoonzero) [inline model_zoo/census_model_sqlflow/wide_and_deep/wide_deep_functional_tensor_interface_fc.py:76]: Is `FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY` some data only used for testing? In real cases, does `FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY` should be generated by the `COLUMN` clause? Also the `TRANSFORM_OUTPUTS`.

**Review by typhoonzero — APPROVED** [review]
LGTM++

File: elasticdl_preprocessing/layers/concatenate_with_offset.py [hunk 2]
```diff
@@ -35,6 +35,13 @@ def __init__(self, offsets, axis=-1):
         self.axis = axis
 
     def call(self, inputs):
+        return (
+            self._call_with_none_offsets(inputs)
+            if self.offsets is None
+            else self._call_with_offsets(inputs)
```
Comment (chunyang-wen) [inline elasticdl_preprocessing/layers/concatenate_with_offset.py]: It is better not to write complex clauses in a single line.

Comment (brightcoder01) [reply elasticdl_preprocessing/layers/concatenate_with_offset.py]: Good suggestion, updated.

File: model_zoo/census_model_sqlflow/wide_and_deep/wide_deep_functional_tensor_interface_fc.py (hunk 1 above)
Comment (brightcoder01) [reply model_zoo/census_model_sqlflow/wide_and_deep/wide_deep_functional_tensor_interface_fc.py:76]: FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY and TRANSFORM_OUTPUTS won't exist in the generated code.
I think it's the intermediate object during code generation. It should be exist in GO.

**Pull request merged**